src/content-handlers/iiif/extensions/uv-aleph-extension/config/config.json

```diff
@@ -199,6 +199,7 @@
         "clickToZoomEnabled": "Mouse Click To Zoom",
         "pagingEnabled": "Two Page View",
         "reducedMotion": "Reduce motion (disables animations)",
+        "truncateThumbnailLabels": "Truncate Thumbnail Labels",

```

Comment:
Any idea why this file includes raw text instead of placeholder strings like `$truncateThumbnailLabels`? Is this something that needs to be fixed in a follow-up PR? (I'm not sure who actually uses this extension at this point...)